scalableminds/webknossos
# move to Google Analytics 4

Migrates the Google Analytics integration to version 4. Implements IP-anonymization and migrates existing event tracking.

### Steps to test:
- set config `googleAnalytics.trackingId = "G-KMSSTEQ438"`
- check Google Analytics dashboard:
<img width="1166" alt="Screenshot 2022-02-08 at 14 31 32" src="https://user-images.githubusercontent.com/335438/152997254-eb026414-0b30-43cc-bc69-546dc3ccbadd.png">


### Issues:
- fixes #5439 

------
(Please delete unneeded items, merge only when none are left open)
